exla/lib/exla.ex

```diff
@@ -133,31 +133,16 @@ defmodule EXLA do
   That's because when the Erlang VM runs as root, it has to manage
   all child programs.
 
-  At the same time, Google XLA's shells out to child program during
-  compilation and it must retain control over how child programs
-  terminate.
+  At the same time, Google XLA's shells out to child program and
+  must retain control over how child programs terminate.
 
-  To address this, simply make sure you wrap the Erlang VM in
-  another process, such as the shell one. In other words, if you
-  are using releases, instead of this:
+  You can pass the `--init` flag to `docker run`, so it runs an
+  `init` inside the container that forwards signals and reaps
+  processes.
 
-      RUN path/to/release start
-
-  do this:
-
-      RUN sh -c "path/to/release start"
-
-  If you are using Mix inside your Docker containers, instead of this:
-
-      RUN mix run
-
-  do this:
-
-      RUN sh -c "mix run"
```

Comment:
Should we keep this but say `CMD` instead of `RUN`?

Reply:
I think so too!